macros/schema_tests/not_null_proportion.sql

```diff
@@ -1,5 +1,5 @@
 {% macro test_not_null_proportion(model) %}
-  {{ return(adapter.dispatch('test_not_null_proportion', packages = dbt_utils._get_utils_namespaces())(model, **kwargs)) }}
+  {{ return(adapter.dispatch('test_not_null_proportion', macro_namespace = 'dbt_utils')(model, **kwargs)) }}
```

Comment:
This is fine. You can also have this just be `'dbt_utils'` (excluding `macro_namespace` keyword arg name) for parity with other macros.